src/box/index.h

```diff
@@ -459,23 +459,23 @@ key_validate(const struct index_def *index_def, enum iterator_type type,
 	     const char *key, uint32_t part_count);
 
 /**
- * Check that the supplied key is valid for a search in a unique
- * index (i.e. the key must be fully specified).
+ * Check that the supplied primary index key is valid for a search in
```

Comment:
That is not necessary a primary index, so you should not change the comment.

Reply:
Fixed. See diff below.